financial-templates-lib/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Arbitrage automation and libraries for UMA financial templates",
   "private": true,
   "dependencies": {
+    "node-fetch": "^2.6.0",
```

Comment:
Why did you decide to use `node-fetch`? I've had great experience with `axios` in the past, which handles the intermediate `.json()` step better plus also handles errors better. More here: https://medium.com/@jeffrey.allen.lewis/http-requests-compared-why-axios-is-better-than-node-fetch-more-secure-can-handle-errors-better-39fde869a4a6

Reply:
I've also used axios but node-fetch is waaay smaller (441B vs 13.1kB) so for now there is no real value add for axios. can always change this later if we need additional error handling. other big pro is node-fetch is easy with asyinc functions

Reply:
ok, I'm just worried about situations where we return with status=200 but we don't actually get any gasprices in the JSON. Maybe add a check in `getPrice()`  that there is actually a `json.fast` property, and if not use the default value

Reply:
ye, this is a good point. I will put a check to ensure that the object returned indeed contains what we need before returning.